doc/examples/client_bulk.rst

```diff
@@ -181,7 +181,7 @@ For example, the fourth and fifth write operations below get executed successful
     'idx': 2,
     'code': 11000,
     'errmsg': 'E11000 duplicate key error ... dup key: { _id: 5 }', ...
-    'op': {'insert': 'db.test_five', 'document': {'_id': 5}}}]
+    'op': {'insert': 0, 'document': {'_id': 5}}}]
```

Comment:
Interesting, do we have any tests that asserts we send the correct bulkWrite format here?

Reply:
I can add a test for that!